src/test/java/core/basesyntax/strategy/OperationHandlerStrategyTest.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import core.basesyntax.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.impl.OperationHandlerStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.strategy.impl.SupplyOperationHandler;
+import core.basesyntax.transaction.FruitTransaction;
+import core.basesyntax.transaction.Operation;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class OperationHandlerStrategyTest {
+    private OperationHandlerStrategy strategy;
+
+    @BeforeEach
+    void setUp() {
+        Map<Operation, OperationHandler> strategies = new HashMap<>();
+        strategies.put(Operation.BALANCE, new BalanceOperationHandler());
+        strategies.put(Operation.SUPPLY, new SupplyOperationHandler());
+        strategies.put(Operation.RETURN, new ReturnOperationHandler());
+        strategies.put(Operation.PURCHASE, new PurchaseOperationHandler());
+        strategy = new OperationHandlerStrategyImpl(strategies);
```

Comment:
Do you actually need to init it before each test? Maybe it will be better to init it **before all**?